spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -203,6 +205,31 @@ There is one `MetricRegistry` instance for each of the scopes listed in <<scopes
 Metrics can be added to or retrieved from the registry either using the `@Metric` annotation
 (see <<api-annotations, Metrics Annotations>>) or using the `MetricRegistry` object directly.
 
+[[reusing_of_metrics]]
+==== Reusing of Metrics
+
+By default it is not allowed to register more than one metric under a certain name in a scope. This is done
+to prevent hard to spot copy & paste errors, where for example all methods of a Jax-Rs class are marked with
+`@Timed(name="myApp", absolute=true)`.
+
+If this behaviour is required, then it is possible to mark all such instances as _reusable_ by passing
+the respective flag via Metadata or field in the Annotation. Gauges are not reusable.
+
+TIP: If you want to re-use a metric name, then you need to also explicitly set the `name` field.
+
```

Comment:
see my other comment on this.  I may also have 2 methods in 2 different classes try to register with the same name if I set absolute=true.